### PR TITLE
Support multiple choice datasets

### PR DESCRIPTION
**Pull request by norabelrose**

### CODE REVIEW

**Review by lauritowal — APPROVED** [review]

File: elk/training/classifier.py [hunk 1]
```diff
@@ -85,7 +84,9 @@ def fit(
         num_classes = self.linear.out_features
         loss_fn = bce_with_logits if num_classes == 1 else cross_entropy
         loss = torch.inf
-        y = y.float()
+        y = y.to(
```
[the remark this answers is not in the document]
Comment (norabelrose) [reply elk/training/classifier.py:87]: `F.cross_entropy` wants integer labels whereas `F.binary_cross_entropy_with_logits` wants float labels, they're inconsistent
